### ISSUE TITLE
Support invisible nodes

### ISSUE THREAD

**Issue by Sponge5** [hbmartin/graphviz2drawio#67]
Hi, I'm getting KeyError on both invisible nodes in my graph. Seems that graphviz doesn't create those nodes when converting to SVG.

Here's the dot source and I'm adding the generated SVG by hand as an attachment as well. (`dot -Gsplines=none anon.dot | neato -n -Gsplines=ortho -Tsvg -oanon.svg`)

```
digraph G {
    graph [rankdir = LR, splines=ortho];

    node[shape=record];

    emmc [label="eMMC" color="blue"]
    boot_host [label="" style=invis width=0 height=2]
    spinor [label="SPI NOR" color="green"]
    soc [label="SoC" height=4]
    nand [label="Raw NAND" color="yellow"]
    dev_eth [label="" style=invis width=0 height=2]
    eeprom [label="I2C EEPROM" color="red"]

    emmc -> soc [arrowhead=none]
    boot_host -> soc [xlabel="Boot Source"]
    boot_host -> soc [xlabel="USB" dir=both]
    spinor -> soc [arrowhead=none]
    soc -> nand [arrowhead=none]
    soc -> dev_eth [xlabel="USB-OTG" dir=both]
    soc -> dev_eth [xlabel="Ethernet" dir=both]
    soc -> eeprom [arrowhead=none]
}
```
![anon](https://github.com/hbmartin/graphviz2drawio/assets/36312261/a8bd4011-1b6a-48fd-ba7e-e3297c13304d)

**Comment by hbmartin** [comment]
I started work on this but it will be more effort than I anticipated so will go out with 0.4 release
If you're interested I could use the help 😄 https://github.com/hbmartin/graphviz2drawio/tree/invisible-nodes

**Comment by hbmartin** [comment]
The fix for this will ship in 0.4 tomorrow, please reopen this issue if it doesn't meet your needs.

<img width="397" alt="Screenshot 2024-07-07 at 3 04 23 PM" src="https://github.com/hbmartin/graphviz2drawio/assets/745166/96b871b1-4841-4eb5-9f0d-bcf8c60db970">
